### PR TITLE
feat(build): add arm64 support

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

**Comment by zlesnr** [comment]
Relates to #774

File: scripts/install.sh [hunk 1]
```diff
@@ -22,6 +22,18 @@ else
 	exit 1
 fi
 
+
+if [ "$(uname -m)" == "x86_64" ]; then
+    MACHINE="x86_64"
```
Comment (sanderblue) [inline scripts/install.sh:27]: Should we set this variable at the top of the file with `x86_64` as the default? Seems to be a prevalent pattern in the make files.

File: scripts/install.sh [hunk 2]
```diff
@@ -22,6 +22,18 @@ else
...
+elif [ "$(uname)" == "aarch64" ]; then
+    MACHINE="aarch64"
+else
+	echo "This machine architecture is not supported."
```
Comment (sanderblue) [inline scripts/install.sh:31]: Maybe we list the supported architectures for clarity? 

> "This machine architecture is not supported. The supported architectures are x86_64 and aarch64."

File: scripts/install.sh [hunk 3]
```diff
@@ -14,12 +14,21 @@ echo "Starting installation."
...
+    exit 1
+fi
+
+if [ "$(uname -m)" == "x86_64" ]; then
+    MACHINE="x86_64"
+elif [ "$(uname)" == "aarch64" ]; then
+    MACHINE="aarch64"
+else
+    echo "This machine architecture is not supported. The supported architectures are x86_64 and aarch64."
+    exit 1
```
Comment (jbeveland27) [inline scripts/install.sh:31]: Just curious - we're building for arm 6/7, but not looking for that machine type here (or at least I don't think we are). Should this be adjusted will this work and I'm just missing something?

Comment (zlesnr) [reply scripts/install.sh:31]: Yeah, I want to see what the archive names are and then I can update the script.

Comment (jbeveland27) [reply scripts/install.sh:31]: Or should we just not build for 32-bit architectures?

Comment (zlesnr) [reply scripts/install.sh:31]: But that will take a release.

Comment (jbeveland27) [reply scripts/install.sh:31]: Cool, makes sense

**Review by jbeveland27 — APPROVED** [review]

**Pull request merged**